### PR TITLE
[ICDS] remove possibility to location modal in the awc_report

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @Rohit25negi @calellowitz 

I remove the possibility to close location modal for the AWC report page.

Ticket: https://dimagi-dev.atlassian.net/browse/ICDS-390

Need QA: No
Environment: ICDS
locally tested: Yes

Regards,
Łukasz

### CODE REVIEW

File: custom/icds_reports/static/js/directives/location-filter/location-filter.directive.js [hunk 1]
```diff
@@ -213,6 +221,7 @@ function LocationFilterController($rootScope, $scope, $location, $uibModal, loca
             templateUrl: 'locationModalContent.html',
             controller: LocationModalController,
             controllerAs: '$ctrl',
+            backdrop: vm.isAwcReport() ? 'static' : true,
```
Comment (calellowitz) [inline custom/icds_reports/static/js/directives/location-filter/location-filter.directive.js]: what does this do?

**Review by calellowitz — COMMENTED** [review]
It looks like you removed the close button, but what happens if someone clicks outside of the modal itself?

File: custom/icds_reports/static/js/directives/location-filter/location-filter.directive.js (hunk 1 above)
Comment (lwyszomi) [reply custom/icds_reports/static/js/directives/location-filter/location-filter.directive.js]: @calellowitz The ```backdrop``` option in the angular-ui modal turn off the possibility to close the modal when you click outside of it. 

"
backdrop (Type: boolean|string, Default: true) - Controls presence of a backdrop. Allowed values: true (default), false (no backdrop), 'static' (disables modal closing by click on the backdrop).
" - https://angular-ui.github.io/bootstrap/#!#modal

Comment (calellowitz) [reply custom/icds_reports/static/js/directives/location-filter/location-filter.directive.js]: thanks!

**Pull request merged**